app/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping.md

```diff
@@ -37,9 +37,13 @@ to identify which admin value and role name to map from the IdP to {{site.base_g
 The `admin_claim` value specifies which IdP username value should map to Kong Manager. 
 The username and password are required for the user to log into the IdP.
 
-The `authenticated_groups_claim` value specifies which IdP claim should be assigned to the
-specified {{site.base_gateway}} user. The exact value depends on your IdP -- for example, Okta 
-configures claims for `groups`, and other IdPs might configure them as `roles`.
+The `authenticated_groups_claim` value specifies which IdP claim should be used to assign {{site.base_gateway}} roles to the
+specified {{site.base_gateway}} admin. 
+
+- The group claim name depends on your IdP -- for example, Okta configures claims for `groups`, and other IdPs might configure them as `roles`. 
+- The group claim value need to follow the format `<workspace_name>:<role_name>` for dynamic role assigning. 
+
+For example, if we set `"authenticated_groups_claim": ["groups"]` and `groups:["default:super-admin"]` is returned on admins' id tokens, these admins gets `super-admin` role in `default` workspace. 
```

Comment:
@liyangau I think there's some confusion here (which I've also helped create in the original). The _value_ of `authenticated_groups_claim` needs to be something like `groups` (for Okta) or `roles` for a different IdP.

But if I'm understanding correctly, the `<workspace_name>:<role_name>` format is required in the IdP, not in the config for Kong Manager. I ignored this fact, and your changes don't make it quite clear. I can edit for clarity, but can you verify my understanding first? Thanks so much, and apologies for the original confusion!

Reply:
Hi Jennifer,

Thanks for taking a look. Your understanding is correct, `<workspace_name>:<role_name>` needs to be the value set on the IdP side. 

I though ```groups:["default:super-admin"]` is returned on admins' id tokens```  +  `The group claim value need to follow the format <workspace_name>:<role_name> for dynamic role assigning.`  are sufficient to suggests that `<workspace_name>:<role_name>` need to be set on IdP side and returned in `id_token` but maybe it is not clear enough?

I am happy for you to change any wording to make it clearer.

The main purposed of this PR is to let the customer know what they need to set on the IdP side in order to use dynamic role assigning. 

Thank you.